InGame/Overlay/OverlayManager.cs

```diff
@@ -457,6 +470,16 @@ private void SetState(MenuState newState)
             _currentMenuState = newState;
         }
 
+        private static void HandleGameScaleChanged(GameScaleDirection scaleDirection)
```

Comment:
There are no other methods with naming convention "Handle...".  Suggest renaming to match legacy author's conventions and leave "Handle..." for future event handlers, if needed.

Reply:
Good suggestion. Updated here: https://github.com/ihm-tswow/Links-Awakening-DX-HD/pull/9/commits/7194ce95357a8f5141dfeac2fcb1fe8d64a0bd81